backend/inspirehep/disambiguation/tasks.py

```diff
@@ -218,15 +229,43 @@ def _get_not_disambiguated_authors(authors):
             yield author
 
 
+def find_stub_author_in_matched_authors(matched_authors_references):
+    for author_reference in matched_authors_references:
+        author = AuthorsRecord.get_record_by_pid_value(
```

Comment:
maybe it woould be better to use `get_records_by_pids` to avoid doing N queries here, or even push the stub filtering to the DB query?

Reply:
Yes, with stub filtering it might be worth doing one db query